README.md

```diff
@@ -30,13 +30,27 @@ Lean (version 3.3.1, nightly-2018-04-10, commit d36b859c6579, Release)
 
 # Installation
 
-**Linux/OS X/Cygwin/MSYS2/git bash/...**: Run the following command in a terminal:
```

Comment:
I think we should keep OS X in this list, not everyone is using Homebrew (feel free to change to "macOS" though). The new subsection should perhaps be called "Homebrew" then.